test/test_suite.py

```diff
@@ -43,7 +43,7 @@
 from sqlalchemy import LargeBinary
 from sqlalchemy import Boolean
 from sqlalchemy import String
-from sqlalchemy.types import Integer
+from sqlalchemy.types import Integer, Text
 from sqlalchemy.types import Numeric
```

Comment:
nit: can merge this also above if merging multiple, else keeping each separate is okay to be consistent across files.